dateutil/test/test_tz.py

```diff
@@ -1908,11 +1910,28 @@ def testFilestreamWithNameRepr(self):
         tzc = tz.tzfile(fileobj)
         self.assertEqual(repr(tzc), 'tzfile(' + repr('foo') + ')')
 
+    @pytest.mark.skipif(not SUPPORTS_PARTIAL_MINUTES, reason='Partial minute offset supported')
+    def testPartialMinuteSupport(self):
+        # If user running python 3.6 or newer, exact offset is used
+        tzc = tz.tzfile(BytesIO(base64.b64decode(EUROPE_HELSINKI)))
+        self.assertEqual(str(datetime(1900, 1, 1, 0, 0, tzinfo=tzc)),
```

Comment:
I think I would prefer a test that looks like this:

```python
assert datetime(1900, 1, 1, 0, 0, tzinfo=tzc).utcoffset() == timedelta(hours=1, minutes=39, seconds=52)
```

Though I am a bit torn because I will admit that the "compare the string" version sure is easy to read.

Reply:
The original was with string (see testRoundNonFullMinutes), but I can change them both to timedelta. 

However, note that the two would test slightly different things (`assert datetime(1900, 1, 1, 0, 0, tzinfo=tzc).utcoffset() == timedelta(hours=1, minutes=39, seconds=52)` tests the date offset through utcoffset function, while the current version tests just the date formation from specified offset)

Reply:
I'll make the change, but in separate commit for easy undo in case we decide to go with original method